websites/T/timolia/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Timolia",
```

Comment:
Service name and folder name must be the same (caps included)

Reply:
fixed